packages/flutter/lib/src/material/menu_anchor.dart

```diff
@@ -1856,11 +1832,15 @@ class SubmenuButton extends StatefulWidget {
 }
 
 class _SubmenuButtonState extends State<SubmenuButton> {
-  FocusNode? _internalFocusNode;
+  late final Map<Type, Action<Intent>> actions = <Type, Action<Intent>>{
```

Comment:
Whoops, thanks for catching! 😅

Reply:
Oh actually, because this references this, doesn't this need to be late?
<img width="533" alt="image" src="https://github.com/user-attachments/assets/1c09be1d-47bf-4417-ab9c-4f8caff646fb">

Reply:
Ahh, yes, it probably does.  I was just thinking that if it has an initial value, it doesn't need to be late.